spec/generator/target_spec.rb

```diff
@@ -50,7 +50,7 @@ module Generator
           target = double("Target").as_null_object
 
           allow(target).to receive(:include_files).and_return(nil)
-          allow(target).to receive(:exclude_files).and_return('**/*.swift')
+          allow(target).to receive(:exclude_files).and_return(['**/*.swift'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.